app/uploaders/avatar_uploader.rb

```diff
@@ -0,0 +1,51 @@
+# encoding: utf-8
+
+class AvatarUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+   include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
+  # storage :fog
+
+  # Override the directory where uploaded files will be stored.
+  # This is a sensible default for uploaders that are meant to be mounted:
+  #def store_dir
```

Comment:
Incorrect indentation detected (column 2 instead of 4).<br>Missing space after #.